lib/skate_web/templates/layout/app.html.eex

```diff
@@ -5,9 +5,7 @@
     static_href = fn conn, path -> apply(static_href_module, static_href_fn, [conn, path]) end
   %>
   <head>
-    <%= if @google_tag_manager_id do %>
-      <%= render "_google_tag_manager.html", assigns %>
-    <% end %>
+
```

Comment:
**question:** is the newline necessary? Getting rid of it would look slightly nicer IMO and save us an extra byte while we're at it.

Reply:
Oops I hadn't realized I left that in, thanks!

Reply:
[`1479175` (#2036)](https://github.com/mbta/skate/pull/2036/commits/1479175801405439e9b637a0c56773f8f0573908)